grandine/src/grandine_config.rs

```diff
@@ -110,7 +109,7 @@ impl GrandineConfig {
             storage_config.eth1_db_size.to_string_as(true),
         );
 
-        info!("Eth1 RPC URLs: [{}]", eth1_rpc_urls.iter().format(", "));
+        info!("Eth1 RPC URLs: {:?}", eth1_rpc_urls);
```

Comment:
This change is required when switching from the `log` to the `tracing` crate, because otherwise a panic ensues 
```
thread 'main' panicked at /home/zarathustra/.cargo/registry/src/index.crates.io-6f17d22bba15001f/itertools-0.13.0/src/format.rs:95:21:
Format: was already formatted once
```

This is important to note, because this is not a compilation but a runtime error, which entails that in case of this being an issue elsewhere - meaning where `log` has been exchanged for `tracing` - the invocation of any of the associated macro's may cause the program to panic.

This leads me to ponder the question how we can best ensure this doesn't occur. There are cumbersome ways of achieving this, by writing individual tests for modules, or by gathering all log messages through the execution of the test suite (i.e. file, line, level, message) and ensuring that all those present in the code are found in the logs. A better approach seems to me to assess the test coverage (as in which lines are of the code are ran during execution of the test suite) and then making sure that all lines with tracing macro's have been covered; this would mean they must have logged, and in case of a panic the test would not have passed.